docs/security.html

```diff
@@ -416,7 +448,10 @@
                 the user for client connections. In this example, clients connect to the broker as user <i>alice</i>.
             </li>
             <li>Pass the JAAS config file location as JVM parameter to each client JVM:
-                <pre>    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre></li>
+                <pre>    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre>
```

Comment:
@ijuma Thank you for the review. There is no real reason to use the static JAAS config file instead of the property for client configuration. But, for the documentation, JAAS config file format is more readable since it is multi-line. And it matches the equivalent broker configuration. I was hoping that specifying the property after the JAAS config file makes it obvious that the property is just a squashed version of the JAAS config file format. Does that make sense?